azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -383,6 +389,119 @@ def __init__(self, cmd):
         self.instances = Instances(self.cmd)
         self.iotops_mgmt_client = self.instances.iotops_mgmt_client
         self.ops: "DataflowEndpointOperations" = self.iotops_mgmt_client.dataflow_endpoint
+        self.instances = Instances(self.cmd)
+
+    def create(
+        self,
+        name: str,
+        instance_name: str,
+        resource_group_name: str,
+        endpoint_type: DataflowEndpointType,
+        show_config: Optional[bool] = None,
+        **kwargs
+    ) -> dict:
+        self.instance = self.instances.show(name=instance_name, resource_group_name=resource_group_name)
```

Comment:
If there is no other use for the instance dict, `self.instances.get_ext_loc(...)` gets you the extended location obj (as well as asserting the instance exists).